Part_1/Calculator/app/src/main/java/com/ensf409/calculator/MainActivity.java

```diff
@@ -43,7 +45,10 @@ protected void onCreate(Bundle savedInstanceState) {
 
     public void setupView()
     {
+        final Calculator calc = new Calculator();
+
     	valueTextView = (TextView)findViewById(R.id.ValueTextView);
+        valueTextView.setText("");
```

Comment:
Instead of setting it in the Java code let's set the initial value in the editor.